src/components/new-request/search.vue

```diff
@@ -607,6 +709,25 @@ export default class Search extends Mixins(CommonMixin, NrAffiliationMixin) {
     this.business = business
     this.entity_type_cd = this.business?.legalType || null
     this.setCorpNum(business?.identifier || null)
+    this.setEntityTypeCd(this.business?.legalType)
+
+    if (this.isConversion) {
+      if (this.business) {
+        // set the from business for alteration (conversion)
+        this.setOriginEntityTypeCd(this.business.legalType)
+        // special case if the from business is BEN
+        // set conversionType & entity_type_cd because there's only one alteration type for it
```

Comment:
Good comment, thanks!

I'll test the "BEN" error as soon as the CI-PR build finishes.

Reply:
Nice! This change fixed both issues I found (designation + BEN). 🎉 

![image](https://github.com/bcgov/namerequest/assets/10002889/ca23e2f0-8987-4c93-8324-50f8466d20f7)

Reply:
(This conversation can be resolved.)